src/main.rs

```diff
@@ -234,16 +234,26 @@ fn create(command: args::CreateCommand) -> anyhow::Result<()> {
 
     packages.extend(constants::AUR_DEPENDENCIES.iter().map(|s| String::from(*s)));
 
+    let pacman_conf_path = command
+        .pacman_conf
+        .unwrap_or_else(|| "/etc/pacman.conf".into());
+
     info!("Bootstrapping system");
     pacstrap
         .execute()
+        .arg("-C")
+        .arg(&pacman_conf_path)
         .arg("-c")
         .arg(mount_point.path())
         .args(packages)
         .args(&command.extra_packages)
         .run()
         .context("Pacstrap error")?;
 
+    // Copy pacman.conf to the image.
```

Comment:
Doesn't this mean that you can have a clean config file?

Reply:
Hmm I don't know if I get what you mean? This command copies the pacman.conf that was used during installation onto the image, so that the image is using the same configuration.

Reply:
I guess that makes sense